### DISCUSSION TITLE
Meow vs Kakoune.el

### DISCUSSION THREAD

**Discussion by gtusr** [meow-edit/meow#371] [Q&A]
Any chance someone could write an in-depth comparison between these two modal editing packages since they are both based on an incomplete implementation of the ideas expressed on the [kakoune website?](https://kakoune.org)

Would also be helpful to know how meow.el relates to packages like expand-region.el and multiple-cursors.el.  Are they still useful in combination with meow or rendered redundant by it?

Any reason to still use hydra.el or transient.el with meow or does that latter already subsume these packages as well?

Would also be interested in hearing any opinions from users who have tried using both of these packages and have found any quirks that are worth warning others about.

**Comment by DogLooksGood** [comment]
Since I'm the author, my opinion is not objective enough. Meow is not going to implement a kak in Emacs, kak is one of the inspirations of Meow.  Before Meow, I used both expand-region and easy-kill to enhance selecting, and used both multiple-cursors and iedit to enhance batch editing.  With Meow, I think all these 4 packages are no longer needed. 

Meow has no opinion about these key binding packages, like hydra, transient and general. Personally, I prefer builtin `define-key`.
